Done/exp

Pramell94 opened the pull request:
Basic error handling for missing exp files or none exp files
Updated graphs to take multiple results
Edited name on result files to use _ instead of BLANKSPACE

I come in peace please, I mean no harm, I beg <3

JoelScarinius reviewed (changes requested):
You need to run git pull origin main from this branch it is 39 commits behind main and therefore it is failing 2 workflows.

JoelScarinius commented:
Seems to be some lintong erros causing crashes. In vscide you can go to a file and choose convert indentation to tabs and this should fix this error. There are 2 curly brases in the bottom of the vscod window where you can press to access this option.

JoelScarinius commented:
I thought Linus disabled this rule

Pramell94 commented:
Fixed the problem with the GUI that it couldn't be run more than one time, with a well placed pygame.init()
Fixed 1 trillion linter problem and rules that master linter giga ruler didn't agree with
Fixed a problem where the program would crash if exp_done folder was missing, Now creates it if missing

JoelScarinius reviewed (approved):
Well done fixing these issues!
